docs/app/partials/home.tmpl.html

```diff
@@ -63,7 +63,8 @@ <h2 id="long-term-support" class="md-headline">Long Term Support</h2>
       AngularJS Material has entered a Long Term Support (LTS) period which ends on December 31st,
       2021. This date aligns with the
       <a href="https://docs.angularjs.org/misc/version-support-status" target="_blank" rel="noopener">
-        AngularJS LTS</a> period.
+      AngularJS LTS</a> period. Explore End Of Life (EOL) options
+      <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429">here</a>.
```

Comment:
```suggestion
      <a href="https://blog.angular.io/finding-a-path-forward-with-angularjs-7e186fdd4429" target="_blank" rel="noopener">here</a>.
```
This should use these attributes similar to the link above.